README.md

```diff
@@ -1,89 +1,248 @@
 Svalinn plugins and command extensions for Trelis
-===================================================
+=================================================
 
 **Beta:** This software is currently under early development.  It has been
 demonstrated to work on a wide range of problems, but the build system is not
 well developed.
 
-Requirements
-============
+Prerequisites
+=============
 
-- Trelis 16.x SDK
-- Armadillo
-- MOAB (and therefore HDF5 & Lapack/blas)
-- DAGMC
+In order to build the plugin, you must have access to Trelis-16.5 and the
+Trelis-16.5 SDK. Additionally, the following system packages must be present on
+your computer:
 
-Build
-======
+* BLAS/LAPACK
+* Armadillo
```

Comment:
There should no longer be any dependency on BLAS/LAPACK or Armadillo - only Eigen3

Reply:
I forgot about that. I removed all references to blas/lapack and armadillo